src/webgpu/shader/validation/shader_io/interpolate.spec.ts

```diff
@@ -0,0 +1,68 @@
+export const description = `Validation tests for the interpolate attribute`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+import { generateShader } from './util.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+// List of valid interpolation attributes.
+const kValidInterpolationAttributes = [
+  '',
+  '[[interpolate(flat)]]',
+  '[[interpolate(perspective)]]',
+  '[[interpolate(perspective, center)]]',
+  '[[interpolate(perspective, centroid)]]',
+  '[[interpolate(perspective, sample)]]',
+  '[[interpolate(linear)]]',
+  '[[interpolate(linear, center)]]',
+  '[[interpolate(linear, centroid)]]',
+  '[[interpolate(linear, sample)]]',
+] as const;
+
+g.test('parameters')
```

Comment:
This test name doesn't really seem to match the fact that this is testing parameter **interpolation decorations**.

Reply:
Changed the name. The full path is now:
`shader,validation,shader_io,interpolate,type_and_sampling`